lms/static/js/demographics_collection/DemographicsCollectionModal.jsx

```diff
@@ -368,11 +404,13 @@ class DemographicsCollectionModal extends React.Component {
             }
           </Wizard.Page>
           <Wizard.Closer>
-            <h3>Thank you! You’re helping make edX better for everyone.</h3>
+            <h3>
+              {gettext("Thank you! You’re helping make edX better for everyone.")}
+            </h3>
           </Wizard.Closer>
           <Wizard.ErrorPage>
             <div>
-              {this.state.error.length ? this.state.error : "something went wrong"}
+              {this.state.error.length ? this.state.error : gettext("something went wrong")}
```

Comment:
I added the gettext call here as I figured this would, eventually, have a error message other than "something went wrong". I can always remove this if this is not the case.